templates/web-deployment.yaml

```diff
@@ -63,17 +63,21 @@ spec:
             - name: CONCOURSE_CLUSTER_NAME
               value: {{ .Values.concourse.web.clusterName | quote }}
             {{- end }}
-            {{- if .Values.concourse.web.maxConns }}
-            - name: CONCOURSE_MAX_CONNS
-              value: {{ .Values.concourse.web.maxConns | quote }}
+            {{- if .Values.concourse.web.apiMaxConns }}
+            - name: CONCOURSE_API_MAX_CONNS
+              value: {{ .Values.concourse.web.apiMaxConns | quote }}
+            {{- end }}
+            {{- if .Values.concourse.web.backendMaxConns }}
+            - name: CONCOURSE_BACKEND_MAX_CONNS
+              value: {{ .Values.concourse.web.backendMaxConns | quote }}
             {{- end }}
             {{- if .Values.concourse.web.enableGlobalResources }}
             - name: CONCOURSE_ENABLE_GLOBAL_RESOURCES
               value: {{ .Values.concourse.web.enableGlobalResources | quote }}
             {{- end }}
-            {{- if .Values.concourse.web.enableLidar }}
```

Comment:
some flags like this were deprecated - should I include something about it in CHANGELOG.md?

Reply:
Probably wouldn't hurt - we'll try to be thorough with the GitHub release notes, but chart consumers might forget to read those.

Reply:
@pivotal-jamie-klassen, I just submitted https://github.com/concourse/concourse-chart/pull/59/commits/10e19b205ae512dd9d327cc76331678a885a529c to update the changelog 👍